tests/test_integrity.py

```diff
@@ -874,5 +843,5 @@ def test_remove_database(db):
     # Clean up temporary database
     db.session.close()
     db.engine.dispose()
-    if os.path.exists(DB_NAME):
-        os.remove(DB_NAME)
+    # if os.path.exists(DB_NAME):
+    #    os.remove(DB_NAME)
```

Comment:
Is this test/function necessary?  @dr-rodriguez ?

Reply:
Probably not if we've moved it all to conftest. I don't know what happens to the temporary database created, though, but maybe it gets cleaned up automatically after all the tests?

Reply:
The .sqlite file just stays -- which I like. I like being able to look at the test database. (This is also one of the reasons why I added .sqlite to the .gitignore.) Not sure if it needs to be cleaned from memory or something.

Reply:
When it runs as part of github actions it doesn't matter since the container it ran in gets destroyed.   
If you run it locally, the issue is you may not know which version of the file you are using. For example, if the tests fail but you see the sqlite file you may be confused as to how it exists if things failed. It's not a big deal, though, since I think the tests do clear the database prior to starting.